src/main/java/mate/academy/spring/validation/PasswordValidator.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.spring.validation;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class PasswordValidator implements ConstraintValidator<Password, UserRequestDto> {
+    private static final String PASSWORD_PATTERN =
```

Comment:
Let’s create a constant with regex to check email. Remember about naming. If you use Pattern - make it constant too.

see common mistakes https://mate-academy.github.io/jv-program-common-mistakes/java-spring/security/jv-spring-security-checklist